coderd/notifications/enqueuer.go

```diff
@@ -141,14 +143,16 @@ func (s *StoreEnqueuer) EnqueueWithData(ctx context.Context, userID, templateID
 			//
 			// This is more efficient than fetching the user's preferences for each enqueue, and centralizes the business logic.
 			if strings.Contains(err.Error(), ErrCannotEnqueueDisabledNotification.Error()) {
-				return nil, ErrCannotEnqueueDisabledNotification
+				s.log.Debug(ctx, fmt.Sprintf("not enqueueing: %s", ErrCannotEnqueueDisabledNotification.Error()), slog.F("template_id", templateID), slog.F("user_id", userID), slog.F("method", method))
```

Comment:
Nit: I was thinking more like `slog.Error(err)` since that already contains the error we're concerned with.